tests/WebAuthn_forge/src/FCL_Webauthn.sol

```diff
@@ -18,7 +18,7 @@
 // Code is optimized for a=-3 only curves with prime order, constant like -1, -2 shall be replaced
 // if ever used for other curve than sec256R1
 // SPDX-License-Identifier: MIT
-pragma solidity ^0.8.0;
+pragma solidity ^0.8.13;
```

Comment:
The pragma of this file was set to a version without custom errors, despite the fact the contract uses them. That's why I bump the version.